Hunter/Assets/Scripts/Controller/Controller.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+public class Controller : MonoBehaviour
+{
+
+    public static void HunterControler()
+    {
+        if (Input.GetKeyDown(KeyCode.W))
+        {
+            Debug.Log("Hunter Move Forward");
+        }
+        if (Input.GetKeyDown(KeyCode.S))
+        {
+            Debug.Log("Hunter Move Back");
+        }
+        if (Input.GetKeyDown(KeyCode.A))
+        {
+            Debug.Log("Hunter Move Left");
+        }
+        if (Input.GetKeyDown(KeyCode.D))
+        {
+            Debug.Log("Hunter Move Right");
+        }
+    }
```

Comment:
1. Input.GetKeyDown triggers only on pressing. For example, if I want to constanly move forward, then I need to click "S" a lot of times. Try to use Input.GetKey instead.
2. There is more elegant way to do it - use Input.GetAxis or Input.GetAxisRaw. Firstly You will get rid of four similar code blocks and secondly - it's compatible with a lot of input types ((W,A,S,D), Arrows and even Joysticks)